pkg/sqlmigration/sqlmigration.go

```diff
@@ -86,3 +86,93 @@ func WrapIfNotExists(ctx context.Context, db *bun.DB, table string, column strin
 		return q.Err(ErrNoExecute)
 	}
 }
+
+func GetColumnType(ctx context.Context, tx bun.Tx, table string, column string) (string, error) {
```

Comment:
```suggestion
func GetColumnType(ctx context.Context, bun bun.IDB, table string, column string) (string, error) {
```

There's a bun interface which accepts both transactions and the database. We should use that here.